src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets

```diff
@@ -82,25 +82,12 @@ Copyright (c) .NET Foundation. All rights reserved.
       <_DotNetHostFileName Condition=" '$(OS)' == 'Windows_NT' ">dotnet.exe</_DotNetHostFileName>
     </PropertyGroup>
 
-    <!-- Temporary workaround until the naming is finalized. Translate from TrimMode to Action metadata. -->
-    <ItemGroup>
-      <ManagedAssemblyToLink Condition=" '%(ManagedAssemblyToLink.TrimMode)' != '' ">
-        <Action>%(ManagedAssemblyToLink.TrimMode)</Action>
-      </ManagedAssemblyToLink>
-    </ItemGroup>
-
-    <!-- Map public properties to the task options -->
-    <PropertyGroup>
-      <_TrimmerLinkSymbols Condition=" '$(TrimmerRemoveSymbols)' == 'true' ">false</_TrimmerLinkSymbols>
-      <_TrimmerLinkSymbols Condition=" '$(TrimmerRemoveSymbols)' != 'true' ">true</_TrimmerLinkSymbols>
-    </PropertyGroup>
-
     <Delete Files="@(_LinkedResolvedFileToPublishCandidate)" />
     <ILLink AssemblyPaths="@(ManagedAssemblyToLink)"
             ReferenceAssemblyPaths="@(ReferencePath)"
             RootAssemblyNames="@(TrimmerRootAssembly)"
-            DefaultAction="$(TrimMode)"
-            LinkSymbols="$(_TrimmerLinkSymbols)"
+            TrimMode="$(TrimMode)"
+            RemoveSymbols="$(TrimmerRemoveSymbols)"
```

Comment:
FYI - @joperezr - our runtime tests are going to fail when we uptake this change.

https://github.com/dotnet/runtime/blob/266a5414b43664acdeeeec9697ece7e69148801b/eng/testing/linker/SupportFiles/Directory.Build.props#L5

Reply:
Thanks for the headsup, I'm also expecing one more thing to break our tests which is the TrimMode rename. I'll keep an eye on darc update PRs and fix when we get the update.